packages/app/package.json

```diff
@@ -55,6 +57,7 @@
     "@types/jquery": "^3.3.34",
```

Comment:
jQuery ?? #whatyearisit

Reply:
I think you need a Unix Beard to use that